kbase-extension/static/kbase/js/common/cellComponents/filePathWidget.js

```diff
@@ -627,13 +643,16 @@ define([
                         paramSpec.original.text_options.is_output_name === 1;
                     if (!isOutput) {
                         // if it's not an "output", it's a file
-                        fileParams.push(id);
+                        fileParamIds.push(id);
+                    } else {
+                        outputParamIds.push(id);
                     }
                 }
             });
-            model.setItem('fileParamIds', fileParams);
+            model.setItem('fileParamIds', fileParamIds);
             model.setItem('parameterSpecs', parameterSpecs);
             model.setItem('defaultParams', defaultParams);
+            model.setItem('outputParamIds', outputParamIds);
```

Comment:
Track the output parameter ids now, those are all we care about having duplicate values.